createlinks

```diff
@@ -38,3 +38,15 @@ event_templates('nethserver-antivirus-update', @templates);
 event_actions('nethserver-antivirus-update',
 	      'initialize-default-databases' => '00',
 	      'nethserver-antivirus-init-avdb' => '30');
+
+#
+# proxy-modify event and action
+#
+
+event_templates('proxy-modify', qw(
+    /etc/freshclam.conf
+));
+
+event_actions('proxy-modify', qw(
```

Comment:
After removing the chmod from the action, you can remove also the linked action from `proxy-modify` event.

Reply:
It's not only about the password leak. Freshclam does not start downloading at all and aborts when a password is set and the permissions are not ok. In this case the av db won't be updated.
``` 
[root@server2 ~]# freshclam -v
WARNING: Insecure permissions (for HTTPProxyPassword): /etc/freshclam.conf must have no more than 0700 permissions.
``` 
Do you see another option to set correct permissions for /etc/freshclam.conf?

Found an old [thread](https://bugs.contribs.org/show_bug.cgi?id=3375) about that issue.

Reply:
Uhm it's a shame that the file doesn't come with the right permissions.

We can override it with a template-metadata rather than the chown:
```
mkdir -p /etc/e-smith/templates.metadata/etc
```

Put the content below inside `/etc/e-smith/templates.metadata/etc/freshclam.conf`:
```
UID="root"
GID="root"
PERMS="0600"
```